manifests/cinder-csi-plugin/cinder-csi-controllerplugin.yaml

```diff
@@ -80,6 +80,16 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
+        - name: liveness-probe
+          image: quay.io/k8scsi/livenessprobe:v1.1.0
+          args:
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /var/lib/csi/sockets/pluginproxy/csi.sock
+          volumeMounts:
+            - mountPath: /var/lib/csi/sockets/pluginproxy/
```

Comment:
why we have this large mountpath , can we have like /csi , which u did for nodeplugin

Reply:
I copied this .. if you check definition for `csi-snapshotter` or `csi-provisioner`, all same..